### ISSUE TITLE
[Enhancement][Metadata Utilities] Support placeholders and/or regex for values in auto merge rules.

### ISSUE THREAD

**Issue by HerrKnarz** [HerrKnarz/Playnite-Extensions#99]

**Comment by HerrKnarz** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Support placeholders and/or regex for values in auto merge rules and unwanted items](https://trello.com/c/vg7LDMFq/19-support-placeholders-and-or-regex-for-values-in-auto-merge-rules-and-unwanted-items)

**Comment by Cshrimps** [comment]
If possible, it would be highly appreciated that conditional actions could also support regex

**Comment by HerrKnarz** [comment]
Yeah, when I implement that, it should work there as well. Not sure, when I will try to add that feature, though, since the addon is really complex in that regard already. I'll also probably wait for Playnite 11 for bigger changes now, as that will require massive changes anyway.
